taxonomies/ingredients_processing.txt

```diff
@@ -372,6 +372,45 @@ fr:extrait naturel de, extrait naturel d'
 
 #fr:obtenu par extraction hydrique
 
+en:pressed, extraite par pression
+# en:false:compressed
+de:abgepresste
+es:espremido de
+fr:pressé, pressée, pressées
+#fr:false:à pâte pressée, fromage pressé
+it:spremuto di
+sv:presset
+
+#<en:pressed
+en:pressed with alkali
```

Comment:
that's weird, shouldn't it be "processed with alkali"

Reply:
I don't know, thats what I saw in the ingredients.

Reply:
probably a mistake in the ingredients, let's not add it (unless we do find several products that contain it on the ingredients photo)